### PR TITLE
[Website] Revise for directive language

### LINKED ISSUE #408: [Website] Revise for directive language
As a visitor to the mineral website, I would like it to be more clear what needs to be done vs what must be done.

AC:
- audit for where the copy says "you should do" or "probably do this" to be more directive.

### LINKED ISSUE #409: [Website] audit for broken links
As a visitor to the website, I expect that all links should work.

AC:
- no dead links

### PR DESCRIPTION
**Pull request by ghost**
closes #409
closes #408

### Description
Audits the site for more directive language and broken links.

### Motivation and context
Language must clearly communicate what must be done vs. what should be done. For example, revising "you should" or "you can" to "do x." 

https://wendyjm-patch-1--mineral-ui.netlify.com/

### How to test

1. `cd mineral-ui && npm start`
2. Check x, y, z examples
3. Are there any dead links

### Types of changes
- Bug fix (non-breaking change which fixes an issue)

### Checklist

* [x] Renders and functions properly in [all supported browsers](https://github.com/mineral-ui/mineral-ui#browser-support) [n/a]
* [x] Automated tests written and passing
* [x] [Accessibility](http://webaim.org/intro) and [inclusivity](https://24ways.org/2016/what-the-heck-is-inclusive-design/) considered [n/a]
* [x] Rendering performance (initial load time & 60fps) and [perceived performance](http://blog.teamtreehouse.com/perceived-performance) considered [n/a]
* [x] Documentation created or updated
* [x] Tested in [sandbox](https://github.com/facebookincubator/create-react-app#getting-started) if new component or breaking change [n/a]

### How does this PR make you feel?

fixed it

### CODE REVIEW

**Comment by Mikewaite** [comment]
Hi Wendy, what is this story?

**Comment by brentertz** [comment]
Hi @wendyjm FYI, the `autoFocus` and `restoreFocus` props of `Popover` and `Dropdown` are going away in the near future as well as the Focus management examples.  Please do not spend effort on writing for them.  Thanks!

**Comment by Mikewaite** [comment]
Looks good to me.

File: src/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ type State = {
 };
 
 /**
- * Dropdown provides a transitory list of actions that appears upon user interaction with a trigger.
+ * Dropdown presents a list of options or actions after a user interacts with a trigger.
```
Comment (kylegach) [inline src/Dropdown/Dropdown.js]: Dropdown items are always “actions” (click to do something), never “options” (click to select that item).

**Review by kylegach — APPROVED** [review]
One minor comment, but this looks great! So glad to have you aboard, Wendy.

File: src/Dropdown/Dropdown.js (hunk 1 above)
Comment (wendyjm) [reply src/Dropdown/Dropdown.js]: That was one of the minor details we were trying to resolve yesterday. Thanks for the clarification!

Comment (riley) [reply src/Dropdown/Dropdown.js]: updated.

**Pull request merged**